From a13995fbc7e4548dda7b603bbb117c5fd3d2a936 Mon Sep 17 00:00:00 2001
From: Robert Mustacchi <rm@joyent.com>
Date: Mon, 11 Dec 2017 16:23:56 -0800
Subject: [PATCH] RICHMOND-138 mfg allocation table columns are incorrect
 Reviewed by: Dale Ghent <dale.ghent@joyent.com> Approved by: Cody Mello
 <melloc@joyent.com>

---
 allocations.md | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/allocations.md b/allocations.md
index d1b38af..12fecd9 100644
--- a/allocations.md
+++ b/allocations.md
@@ -15,8 +15,8 @@ which are used as part of SMBIOS and other product naming.
 
 ## Products, SKU, Common Name
 
-| SKU | Product ID | Common Name |
-|-----|------------|-------------|
+| SKU | Common Name | Product ID |
+|-----|-------------|------------|
 | Richmond A | JCP 1101 | 600-0002 |
 | Richmond B | JCP 1201 | 600-0004 |
 | Richmond C | JCP 1102 | 600-0006 |
-- 
2.21.0

